python/setup.py

```diff
@@ -163,6 +163,12 @@ def get_package_data_files():
         "attrs",
         "psutil",
         "typed_ast",
+        "tensorflow==2.1.0",
+        "tflite==2.1.0",
```

Comment:
These are optional deps, and let us move them to extra_require

Reply:
Moved now.